ICRCs/ICRC-123/ICRC-123.md

```diff
@@ -0,0 +1,223 @@
+# ICRC-123: Freezing and Unfreezing Accounts and Principals
```

Comment:
I am generally wondering why principals are needed in this standard? aren't we dealing with accounts only in the context of ICRC-1/2 transfers?